geomstats/datasets/utils.py

```diff
@@ -0,0 +1,85 @@
+"""Loading toy datasets."""
+
+import json
+import os
+
+import geomstats.backend as gs
+
+
+DATA_FOLDER = os.path.join(
+    'geomstats', 'datasets', 'data')
+
+CITIES_PATH = os.path.join(
+    DATA_FOLDER, 'cities/cities.json')
+POSES_PATH = os.path.join(
+    DATA_FOLDER, 'poses/poses.json')
+KARATE_PATH = os.path.join(
+    DATA_FOLDER, 'graph_karate/karate.txt')
+KARATE_LABELS_PATH = os.path.join(
+    DATA_FOLDER, 'graph_karate/karate_labels.txt')
+GRAPH_RANDOM_PATH = os.path.join(
+    DATA_FOLDER, 'graph_random/graph_random.txt')
+GRAPH_RANDOM_LABELS_PATH = os.path.join(
+    DATA_FOLDER, 'graph_random/graph_random_labels.txt')
+
+
+def load_cities():
+    """Load data from data/cities/cities.json.
+
+    Returns
+    -------
+    data : array-like, shape=[50, 2]
+        Array with each row representing one sample,
+        i. e. latitude and longitude of a city.
+        Angles are in radians.
+    name : list
+        List of city names.
+    """
+    data = []
+    names = []
+
+    with open(CITIES_PATH) as json_file:
+        data_file = json.load(json_file)
+
+        for row in data_file:
+            lat_in_radians = row['lat'] / 90 * gs.pi / 2
+            lng_in_radians = row['lng'] / 180 * gs.pi
+            data.append(gs.array([lat_in_radians, lng_in_radians]))
+            names.append(row['city'])
+
+    data = gs.array(data)
```

Comment:
Here's how I would do it using the map function:
```python
names = list(map(lambda x: x['city'], data_file))
data = gs.array(list(map(lamda x:  [x[k] / 180 * gs.pi for k in ['lat', 'lng']], data_file)))
```

Reply:
Done.